easybuild/easyconfigs/r/RDP-Classifier/RDP-Classifier-2.13-Java-17.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'Tarball'
+
+name = 'RDP-Classifier'
+version = '2.13'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://sourceforge.net/projects/rdp-classifier'
+description = """The RDP Classifier is a naive Bayesian classifier that can rapidly and accurately provides taxonomic
+ assignments from domain to genus, with confidence estimates for each assignment."""
+
+toolchain = SYSTEM
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['rdp_classifier_%(version)s.zip']
+checksums = ['d2eb9a989d7e31ff824de6876bc1d8ab70d97112ece167afe8d87a3cd412086a']
+
+dependencies = [('Java', '17')]
```

Comment:
`Java/11` would've probably been a better choice here, since then it can be used as a dep in easyconfigs using `foss/2021b` as toolchain.

Are we even sure if `RDP-Classifier` 2.13 actually supports Java 11.x?

@ItIsI-Orient Can you check if there's an easy way to test that?

Reply:
And we already have `RDP-Classifier-2.13-Java-11.eb`.

So we should at least make try and make sure that `RDP-Classifier` 2.13 indeed works with Java 17.x (and if not, retract this easyconfig)